pgmpy/continuous/discretize/rounding.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import division
+
+import numpy as np
+
+from pgmpy.factors import ContinuousFactor
+from pgmpy.continuous.discretize import BaseDiscretizer
+
+
+class RoundingDiscretizer(BaseDiscretizer):
+    """
+    This class uses the rounding method for discretizing the
+    given continuous distribution.
+
+    For the rounding method,
+
+    The probability mass in x=[low] is,
+    cdf(x+step/2)-cdf(x)
+
+    The probability mass in x=[low+step, low+2*step, ......... , high-step] is,
+    cdf(x+step/2)-cdf(x-step/2)
+
+    where, cdf is the cumulative density function of the distribution
+    and step = (high-low)/cardinality.
+
+    Examples
+    --------
+    >>> from pgmpy.factors import ContinuousFactor
```

Comment:
@yashu-seth You can also write this as:
```
For the rounding method, the probability mass is:
cdf(x+step/2) - cdf(x)                for x = low
cdf(x+step/2) - cdf(x-step/2)     for low < x < high
```
This is a more standard for writing equations like this.